bluebutton-data-model-rif/src/main/resources/db/migration/V12__Add_field.sql

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
I would rename this file to V15__**** as I have already created files up to V14__***.   They just haven't been merged into production yet.    Also I would make the file name a little bit more descriptive.  Thanks.